README.Rmd

```diff
@@ -87,72 +87,119 @@ on how to improve the process on our [forum](https://discuss.ropensci.org/) and
 
 # <a href="#editors" name="editors"></a> Editors and reviewers
 
-### Associate editors
+rOpenSci's Software Peer Review process is run our team of staff and volunteer editors and reviewers.
```

Comment:
@noamross I'm not sure that "staff and volunteer editors" is the best phrase - please edit if you like. Note that former phrasing did not place general and stats editors equally, which is something I've now tried to do.

Also note that I've removed the term "Associate Editors", so we can explicitly distinguish between *General* and *Stats* editors. Revert or otherwise modify if you think "Associate" is important.

Reply:
I don't think we need any of the modifiers ; associate, general, or stats.  Just "Editors".  I'd remove staff and voluntter too: "rOpenSci's Software Peer Review process is run our team of dedicated editors and reviewers."

Reply:
done